Makefile

```diff
@@ -103,10 +103,10 @@ flake:
 	@echo "flake8 passed"
 
 codespell:  # running manually
-	@codespell --builtin clear,rare,informal,names,usage -w -i 3 -q 3 -S $(CODESPELL_SKIPS) --ignore-words=ignore_words.txt $(CODESPELL_DIRS)
+	@codespell --builtin clear,rare,informal,names,usage -w -i 3 -q 3 -S $(CODESPELL_SKIPS) --ignore-words=ignore_words.txt --uri-ignore-words-list=bu $(CODESPELL_DIRS)
```

Comment:
we have a couple of URLs for Boston University, codespell thinks `bu` -> `by`

Reply:
figured it out. Hopefully this should fix it: https://github.com/GuillaumeFavelier/actions-codespell/pull/2

ping @GuillaumeFavelier